<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>1500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,20,4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_2">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="multigrapf">
         <property name="font">
          <font>
           <family>Proxima Nova Rg</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 108, 217);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 108, 217, 255), stop:1 rgba(0, 170, 255, 255));
color : white;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Мультиграф v0.6&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="time_label">
         <property name="font">
          <font>
           <family>Proxima Nova Rg</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 108, 217);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 108, 217, 255), stop:1 rgba(0, 170, 255, 255));
color : white;</string>
         </property>
         <property name="text">
          <string>Date/Time</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <layout class="QGridLayout" name="gridLayout" rowstretch="10,3" columnstretch="10,3">
       <item row="1" column="0" colspan="2">
        <widget class="QFrame" name="frame_6">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1">
          <item>
           <widget class="QWidget" name="MessagesWidget" native="true">
            <property name="font">
             <font>
              <family>Open Sans</family>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>BlankCursor</cursorShape>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QFrame" name="frame_4">
         <widget class="QWidget" name="bar_1" native="true">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>150</y>
            <width>61</width>
            <height>181</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(85, 170, 0);
</string>
          </property>
         </widget>
         <widget class="QWidget" name="bar_4" native="true">
          <property name="geometry">
           <rect>
            <x>254</x>
            <y>80</y>
            <width>80</width>
            <height>260</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <widget class="QWidget" name="color_2" native="true">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>70</width>
             <height>250</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 202, 128);</string>
           </property>
          </widget>
          <widget class="QWidget" name="shadow_4" native="true">
           <property name="geometry">
            <rect>
             <x>7</x>
             <y>7</y>
             <width>70</width>
             <height>250</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(0, 0, 0, 100);
border-right:15px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 100), stop:1 rgba(255, 255, 255, 0));
border-bottom:15px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 100), stop:1 rgba(255, 255, 255, 0));
border-left:15px solid qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(0, 0, 0, 100), stop:1 rgba(255, 255, 255, 0));
border-top:15px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 100), stop:1 rgba(255, 255, 255, 0));
background-clip:padding;
/* border-radius: 5px;*/</string>
           </property>
          </widget>
          <zorder>shadow_4</zorder>
          <zorder>color_2</zorder>
         </widget>
         <widget class="QCustomPlot" name="customPlot" native="true">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>741</width>
            <height>421</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Open Sans</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <stylestrategy>NoAntialias</stylestrategy>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <widget class="QPushButton" name="timeButton">
           <property name="geometry">
            <rect>
             <x>1240</x>
             <y>0</y>
             <width>40</width>
             <height>30</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 108, 217);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 108, 217, 0), stop:1 rgba(0, 170, 255, 0));
color : white;</string>
           </property>
           <property name="text">
            <string>⟳</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="bar_2" native="true">
          <property name="geometry">
           <rect>
            <x>490</x>
            <y>150</y>
            <width>61</width>
            <height>181</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 255, 255);
</string>
          </property>
         </widget>
         <widget class="QWidget" name="bar_6" native="true">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>80</y>
            <width>80</width>
            <height>260</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <widget class="QWidget" name="color_4" native="true">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>70</width>
             <height>250</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 202, 128);</string>
           </property>
          </widget>
          <widget class="QWidget" name="shadow_6" native="true">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>70</width>
             <height>250</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(0, 0, 0, 100);
border-right:15px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 100), stop:1 rgba(255, 255, 255, 0));
border-bottom:15px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 100), stop:1 rgba(255, 255, 255, 0));
border-left:15px solid qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(0, 0, 0, 100), stop:1 rgba(255, 255, 255, 0));
border-top:15px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 100), stop:1 rgba(255, 255, 255, 0));
background-clip:padding;
</string>
           </property>
          </widget>
          <zorder>shadow_6</zorder>
          <zorder>color_4</zorder>
         </widget>
         <zorder>customPlot</zorder>
         <zorder>bar_1</zorder>
         <zorder>bar_4</zorder>
         <zorder>bar_2</zorder>
         <zorder>bar_6</zorder>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFrame" name="frame_5">
         <widget class="QGraphicsView" name="graphicsView">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>211</width>
            <height>241</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
            <horstretch>100</horstretch>
            <verstretch>100</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string/>
      </property>
      <widget class="QCheckBox" name="showcenterdots">
       <property name="geometry">
        <rect>
         <x>1070</x>
         <y>70</y>
         <width>300</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Точка в серед.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>160</width>
         <height>66</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Open Sans</family>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>logo.jpg</pixmap>
       </property>
      </widget>
      <widget class="QCheckBox" name="showdots">
       <property name="geometry">
        <rect>
         <x>1070</x>
         <y>40</y>
         <width>300</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Отображать точки</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="EcoCheckBox">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>50</y>
         <width>120</width>
         <height>45</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 108, 217);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 108, 217, 255), stop:1 rgba(0, 170, 255, 255));
color : white;</string>
       </property>
       <property name="text">
        <string>Eco</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="ArchiveButton">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>50</y>
         <width>140</width>
         <height>45</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0123, 123, 123, 255), stop:1 rgba(0, 0, 0, 255));</string>
       </property>
       <property name="text">
        <string>Архив</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>50</y>
         <width>140</width>
         <height>45</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0123, 123, 123, 255), stop:1 rgba(0, 0, 0, 255));</string>
       </property>
       <property name="text">
        <string>Сообщен.</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="percentCheckBox">
       <property name="geometry">
        <rect>
         <x>900</x>
         <y>100</y>
         <width>150</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>В процентах</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="arrowscheckBox">
       <property name="geometry">
        <rect>
         <x>900</x>
         <y>40</y>
         <width>150</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Cтрелки</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="autoscalecheckbox">
       <property name="geometry">
        <rect>
         <x>900</x>
         <y>70</y>
         <width>150</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Автомасшаб</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>50</y>
         <width>140</width>
         <height>45</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0123, 123, 123, 255), stop:1 rgba(0, 0, 0, 255));</string>
       </property>
       <property name="text">
        <string>Меню</string>
       </property>
      </widget>
      <widget class="QPushButton" name="WorkButton">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>50</y>
         <width>140</width>
         <height>45</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Open Sans</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0123, 123, 123, 255), stop:1 rgba(0, 0, 0, 255));
</string>
       </property>
       <property name="text">
        <string>Работа</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
